scratchpad/components/todo/bindings/AuthStrategy.ts

```diff
@@ -0,0 +1,17 @@
+import {AUTHENTICATION_STRATEGY} from '@loopback/authentication';
+import {BasicStrategy} from '@passport/basic';
+
+export class AuthStrategy {
+  constructor(
+    private @userRepo() users
+  )
+  name = AUTHENTICATION_STRATEGY; // 'authentication.strategy'
+  value() {
```

Comment:
+1 on this, I have commented that on other pull request but I'm still seeing no return types.. I would expect return types on every method with return value